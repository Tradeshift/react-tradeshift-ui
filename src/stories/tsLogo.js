import React from 'react';
import { storiesOf } from '@kadira/storybook';
import { TsLogo } from '../components';

const stories = storiesOf('TsLogo', module);

stories.addWithInfo('Basic usage', () => <TsLogo />, { inline: true });
