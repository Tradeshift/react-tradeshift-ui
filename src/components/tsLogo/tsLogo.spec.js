import React from 'react';
import { shallow } from 'enzyme';

import TsLogo from './';

describe('TsLogo', () => {
	it('should render correctly', () => {
		const tree = shallow(<TsLogo />);
		expect(tree).toMatchSnapshot();
	});
});
