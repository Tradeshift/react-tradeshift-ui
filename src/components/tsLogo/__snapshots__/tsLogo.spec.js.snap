exports[`TsLogo should render correctly 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <svg
    className="ts-logo"
    viewBox="0 0 161 109">
    <path
        d="M72.5 1h-70C1 1 0 1.6 0 3v18c0 .8.5 1.6 1.4 2 .3 0 .7 0 1 .2H45v83c0 1.3 1 2.2 2.4 2.2h25c1.4 0 2.4-1 2.4-2.2V3c0-1.3-1-2-2.3-2m86 0h-70C87 1 86 1.8 86 3v103.4c0 1.3 1 2 2.4 2h25c1.4 0 2.4-.8 2.4-2v-83h42.6c.4 0 .7 0 1-.2 1-.3 1.4-1 1.4-2v-18c0-1.3-1-2.2-2.3-2.2"
        fillRule="evenodd" />
</svg>,
  "nodes": Array [
    <svg
      className="ts-logo"
      viewBox="0 0 161 109">
      <path
            d="M72.5 1h-70C1 1 0 1.6 0 3v18c0 .8.5 1.6 1.4 2 .3 0 .7 0 1 .2H45v83c0 1.3 1 2.2 2.4 2.2h25c1.4 0 2.4-1 2.4-2.2V3c0-1.3-1-2-2.3-2m86 0h-70C87 1 86 1.8 86 3v103.4c0 1.3 1 2 2.4 2h25c1.4 0 2.4-.8 2.4-2v-83h42.6c.4 0 .7 0 1-.2 1-.3 1.4-1 1.4-2v-18c0-1.3-1-2.2-2.3-2.2"
            fillRule="evenodd" />
</svg>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 2,
      "_context": Object {},
      "_currentElement": <TSLogo />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StatelessComponent {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <svg
          className="ts-logo"
          viewBox="0 0 161 109">
          <path
                    d="M72.5 1h-70C1 1 0 1.6 0 3v18c0 .8.5 1.6 1.4 2 .3 0 .7 0 1 .2H45v83c0 1.3 1 2.2 2.4 2.2h25c1.4 0 2.4-1 2.4-2.2V3c0-1.3-1-2-2.3-2m86 0h-70C87 1 86 1.8 86 3v103.4c0 1.3 1 2 2.4 2h25c1.4 0 2.4-.8 2.4-2v-83h42.6c.4 0 .7 0 1-.2 1-.3 1.4-1 1.4-2v-18c0-1.3-1-2.2-2.3-2.2"
                    fillRule="evenodd" />
</svg>,
        "_debugID": 2,
        "_renderedOutput": <svg
          className="ts-logo"
          viewBox="0 0 161 109">
          <path
                    d="M72.5 1h-70C1 1 0 1.6 0 3v18c0 .8.5 1.6 1.4 2 .3 0 .7 0 1 .2H45v83c0 1.3 1 2.2 2.4 2.2h25c1.4 0 2.4-1 2.4-2.2V3c0-1.3-1-2-2.3-2m86 0h-70C87 1 86 1.8 86 3v103.4c0 1.3 1 2 2.4 2h25c1.4 0 2.4-.8 2.4-2v-83h42.6c.4 0 .7 0 1-.2 1-.3 1.4-1 1.4-2v-18c0-1.3-1-2.2-2.3-2.2"
                    fillRule="evenodd" />
</svg>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <TSLogo />,
}
`;
